src/tests/integration/test_dual_allocators_ohmms_containers.cpp

```diff
@@ -0,0 +1,105 @@
+//////////////////////////////////////////////////////////////////////////////////////
```

Comment:
could yo move this to src/Containers/tests?

Reply:
I'll try, I do think it is an integration test, but its debatable since support for dual space is part of the Ohmms containers API now.